src/ccf/calc_ccf.jl

```diff
@@ -281,6 +281,37 @@ function ccf_1D(λ::A2, flux::A3, #line_list::ALL, #mask_shape1::A3
     return ccf_out
 end
 
+"""
+    `ccf_1D( λs, fluxes; ccf_plan )`
+    Compute the cross correlation functions of spectra with a mask.
+# Inputs:
+- `λs`: 1-d array of wavelengths
+- `fluxes`:  2-d array of fluxes, individual spectra along first dim
+# Optional Arguments:
+- `ccf_plan`:  parameters for computing ccf (BasicCCFPlan())
+# Returns:
+- 2-d array of size (length(calc_ccf_v_grid(plan)), size(flux, 2))
+"""
+function ccf_1D_test(λ::A2, flux::A3,
```

Comment:
Did you mean to have _test here?

Reply:
Oops, no. I inserted that when I was having issues with the Revise module. It should have gotten taken out. Just renamed it and tested it. I'll push the change. Thanks for catching this.